posthog/test/test_migration_0273.py

```diff
@@ -0,0 +1,175 @@
+            detail={
+                "trigger": {
+                    "job_id": "3",
+                    "job_type": "Export historical events V2",
+                    "payload": {},
+                }
+            },
+        )
+        self.create_entry(
+            apps,
+            self.plugin_configs[2].pk,
+            created_at=timezone.now() - timedelta(days=1),
+            activity="export_fail",
+            detail={
+                "trigger": {
+                    "job_id": "3",
+                    "job_type": "Export historical events V2",
+                    "payload": {},
+                    "failure_reason": "Some reason",
+                }
+            },
+        )
+
+        # Case 4: Recently started export
+        self.create_entry(
+            apps,
+            self.plugin_configs[3].pk,
+            created_at=timezone.now() - timedelta(minutes=2),
+            activity="job_triggered",
+            detail={
+                "trigger": {
+                    "job_id": "4",
+                    "job_type": "Export historical events V2",
+                    "payload": {},
+                }
+            },
+        )
+
+        # Case 5: Started export with storage containing it's running
+        self.create_entry(
+            apps,
+            self.plugin_configs[4].pk,
+            created_at=timezone.now() - timedelta(hours=1),
+            activity="job_triggered",
+            detail={
+                "trigger": {
+                    "job_id": "5",
+                    "job_type": "Export historical events V2",
+                    "payload": {},
+                }
+            },
+        )
+        PluginStorage.objects.create(
+            plugin_config_id=self.plugin_configs[4].pk, key="EXPORT_PARAMETERS", value=json.dumps({"id": "5"})
+        )
+
+        # Case 6: Started export with storage containing another that's running
+        self.create_entry(
+            apps,
+            self.plugin_configs[5].pk,
+            created_at=timezone.now() - timedelta(hours=1),
+            activity="job_triggered",
+            detail={
+                "trigger": {
+                    "job_id": "6",
+                    "job_type": "Export historical events V2",
+                    "payload": {},
+                }
+            },
+        )
+        PluginStorage.objects.create(
+            plugin_config_id=self.plugin_configs[5].pk, key="EXPORT_PARAMETERS", value=json.dumps({"id": "7"})
+        )
+
+    def test_migration(self):
+        ActivityLog = self.apps.get_model("posthog", "ActivityLog")  # type: ignore
+
+        entries = ActivityLog.objects.filter(activity="export_fail", is_system=True)
+
+        self.assertEqual(set(entry.detail["trigger"]["job_id"] for entry in entries), {"1", "6"})
+        self.assertEqual(
+            set(entry.detail["trigger"]["failure_reason"] for entry in entries),
+            {"Export was killed after too much inactivity"},
```

Comment:
The reason this doesn't contain "Some reason" is that that wasn't an is_system=True right?

Reply:
Exactly - I'm trying to filter out the rows that were added by the migration in a tricky way.